docs/docs/deployment/hosting/locally-edge-proxy.md

```diff
@@ -229,7 +235,39 @@ Or, log access logs to file in generic format while logging everything else to s
 When adding logger configurations, you can use the `"default"` handler which writes to stdout and uses formatter
 specified by the [`"logging.log_format"`](#logginglog_format) setting.
 
-### `config.json` example
+### Health Check
+
+The health check can be configured depending on the use case for the Edge Proxy by adding the `health_check` object to
+the root of the settings file.
+
+```json
+{
+  ...
+  "health_check": {
+    "count_stale_documents_as_failing": true,
+    "grace_period_seconds": 30
```

Comment:
Are these the new defaults? Looking at https://github.com/Flagsmith/edge-proxy/pull/122/files#diff-4573c040951bc5c2676139dcf260c4a96b0f9b830f15227322c0a1f8545da004R104-R105

In which case these docs should mention how to change the defaults, only if needed